sycl/source/spirv_ops.cpp

```diff
@@ -15,24 +15,26 @@
 // This operation is NOP on HOST as all operations there are blocking and
 // by the moment this function was called, the operations generating
 // the __ocl_event_t objects had already been finished.
-void __spirv_GroupWaitEvents(__spv::Scope Execution, uint32_t NumEvents,
-                              __ocl_event_t * WaitEvents) noexcept {
-}
+SYCL_API void __spirv_GroupWaitEvents(__spv::Scope Execution,
```

Comment:
AFAIK, we don't need to specify GCC attribute for definition - only for declarations. Is it required on Windows?
If not, we can remove them.

Reply:
Yes, that is to make MSVC happy.